client/app/bundles/course/survey/pages/ResponseIndex/index.jsx

```diff
@@ -71,18 +97,36 @@ class ResponseIndex extends React.Component {
     }).isRequired,
   };
 
+  static computeStatuses(responses) {
```

Comment:
@kxmbrian quick question, why the method is defined as static ? for better performance ?